README.md

```diff
@@ -1,13 +1,34 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week my assignment was to create the backend API of Happy Thoughts, a Twitter like app for which I've built a front-end a couple of months ago. I was supposed to set up a database with MongoDB and connect it to my deployed Heroku backend. For this project it was needed to create an API with a possibility to post and get thoughts, but also with the possibility to like thoughts. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+This week I started with creating an endpoint to post thoughts. As soon as I got that one up and running, and could see that my thoughts was being posted properly, I continued with creating a get endpoint to be able to get the existing posts. I sorted them by the time they were created, with the most recent on top. I also limited them to show only the last 20 messages. 
+As soon as I had this working, I set up the endpoint to post likes. Once this was done, I connected the deployed backend to my frontend, which was easy to do by just replacing the URLs already in that repo. 
+To complete the project, I also created one patch endpoint to make it possible yo update a thought, and a delete endpoint to make it possible to delete a thought. 
```

Comment:
sounds like you had a very structured approach and like it was a walk in the park for you! 🤩